src/stylesheets/common/_variables.scss

```diff
@@ -17,7 +17,7 @@
 
 // Display type used for headings, logo, etc.
 $display-font-family: 'Poppins', 'Open Sans', Helvetica, Arial, sans-serif;
-$display-font-weight-bold: 600;
+$display-font-weight-bold: 700;
```

Comment:
oh @souperneon told me to change it to 700. but probably not ready to go to master yet. how do you think @souperneon ?